src/main/res/values-zh/strings.xml

```diff
@@ -0,0 +1,241 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="title_apps">应用</string>
+    <string name="title_settings">设定值</string>
+    <string name="title_shares">分享</string>
+    <string name="title_no_server">没有伺服器</string>
+    <string name="title_upload_list">上载</string>
+    <string name="title_offline_files">离线文件</string>
+    <string name="title_recent_files">最近的文件</string>
+
+    <string name="button_search_application">搜索Google Play</string>
```

Comment:
We have translated Google Play into other languages as well. Please translate it into Chinese also.